automation/features/step-definitions/staking.steps.js

```diff
@@ -47,5 +41,93 @@ When(/^I can see the maximum amount of tokens in my wallet are in the token inpu
   expect(browser.getByTestId('token-amount-input').getValue()).toBe(String(noZeroesFloat))
 })
 
+Then(/^I can see the validator node list$/,()=>{
+expect(browser.getByTestId('validator-node-list')).toBeDisplayed()
+  const nodeList = $(".node-list").$$('[data-testid="node-list-item"]');
+  expect(nodeList.length).toBeGreaterThan(0);
+})
+
+Then(/^the epoch countdown timer is counting down$/,()=>{
+const currentCountdownTimerText = browser.getByTestId('current-epoch-ends-in').getText()
+browser.pause(2100) // let some time pass 
+expect(browser.getByTestId('current-epoch-ends-in').getText()).not.toEqual(currentCountdownTimerText)
+})
+
+Then(/^I pause some seconds$/,()=>{
+browser.pause(600000)
+})
```

Comment:
This looks like an unused step

Reply:
it is used for debugging , i put it in between steps some times when i need to pause the test . will delete and push